components/library/src/pagination/mod.rs

```diff
@@ -262,15 +273,22 @@ mod tests {
         s
     }
 
-    fn create_library(is_index: bool) -> (Section, Library) {
-        let mut library = Library::new(3, 0, false);
-        library.insert_page(Page::default());
-        library.insert_page(Page::default());
-        library.insert_page(Page::default());
+    fn create_library(
+        is_index: bool,
+        num_pages: usize,
+        paginate_reversed: bool,
+    ) -> (Section, Library) {
+        let mut library = Library::new(num_pages, 0, false);
+        for i in 1..num_pages + 1 {
```

Comment:
isn't `...` equivalent to `end + 1`?

Reply:
I looked it up `..=` is an inclusive range. I'll make that change if you prefer,